client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -344,37 +356,37 @@ const mapStateToProps = ( state ) => {
 };
 
 const mapDispatchToProps = {
-	editHomepageAction,
-	writePostAction,
-	addPageAction,
-	manageCommentsAction,
+	trackEditHomepageAction,
+	trackWritePostAction,
+	trackAddPageAction,
+	trackManageCommentsAction,
 	trackEditMenusAction,
 	trackCustomizeThemeAction,
-	changeThemeAction,
+	trackChangeThemeAction,
 	trackDesignLogoAction,
 	trackAnchorPodcastAction,
-	addEmailAction,
-	addDomainAction,
+	trackAddEmailAction,
+	trackAddDomainAction,
 	expand: () => savePreference( 'homeQuickLinksToggleStatus', 'expanded' ),
 	collapse: () => savePreference( 'homeQuickLinksToggleStatus', 'collapsed' ),
 };
 
 const mergeProps = ( stateProps, dispatchProps, ownProps ) => {
-	const { editHomePageUrl, isStaticHomePage, siteSlug } = stateProps;
+	const { isStaticHomePage } = stateProps;
```

Comment:
I know this `mergeProps` was already here, but it seems like a pretty elaborate way just to avoid passing the `isStaticHomePage` prop to the tracking functions. 🤷 I guess it's just personal preference.